objectherkenning_openbare_ruimte/convert_dataset_pipeline/components/convert_dataset.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import os
+import sys
+
+from azure.ai.ml.constants import AssetTypes
+from mldesigner import Input, Output, command_component
+
+sys.path.append("../../..")
+
+from aml_interface.azure_logging import setup_azure_logging  # noqa: E402
+from cvtoolkit.helpers.file_helpers import find_image_paths  # noqa: E402
+
+from objectherkenning_openbare_ruimte.settings.settings import (  # noqa: E402
+    ObjectherkenningOpenbareRuimteSettings,
+)
+
+config_path = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), "..", "..", "..", "config.yml")
+)
+settings = ObjectherkenningOpenbareRuimteSettings.set_from_yaml(config_path)
+log_settings = ObjectherkenningOpenbareRuimteSettings.set_from_yaml(config_path)[

```

Comment:
The settings need to be set only once, then you can do settings["logging"].
Also the class in case has a get function.